docs/serverless/08-01-create-inline-function.md

```diff
@@ -63,7 +63,7 @@ Follows these steps:
 
```

Comment:
Since you've moved the `create-git-function` tutorial, maybe it would be good to add a reference link to it in this doc? It's an open question

Reply:
Sure :)